hadoop-tools/hadoop-azure/dev-support/testrun-scripts/testsupport.sh

```diff
@@ -63,6 +64,35 @@ fnsBlobConfigFileCheck() {
   fi
 }
 
+if ! command -v az &> /dev/null
+then
+  echo "Azure CLI (az) could not be found. Installing Azure CLI..."
+sudo apt update
```

Comment:
This should be under a wait process ? What happens if the installation fails

Reply:
Right. Added a check for it